debian/changelog

```diff
@@ -1,3 +1,14 @@
+libfm-qt (1.1.0-0ubuntu7~ppa2) jammy; urgency=medium
+
+  * Backport to Jammy.
+  * Updated Lubuntu Developers copyright.
+  * Updated symbols file
+  * Removed Repository-Browser field from debian/upstream/metadata.
+  * Removed "-Wl,--as-needed" linker flag.
```

Comment:
Why? Even saying `obsolete` goes a long way. Ditto for the above. :smile:

Reply:
Lintian gripe, removing it seems to work in other stuff and made Lintian calm down. I guess "obsolete" should be put in there.

Reply:
Done.